spec/integration/instance_properties.cpp

```diff
@@ -1,8 +1,4 @@
-struct Point {
-  Point(int x, int y) : x(x), y(y) { }
-
-  int x, y;
-};
+#include "copy_structs.cpp" // Point, Anonymous
```

Comment:
Include a cpp?

Reply:
If this is undesirable I could make both files symlinks of each other, like `inheritance.cpp` and `virtual_override.cpp`.